packages/canvas-tokens/tokens/base.json

```diff
@@ -727,10 +727,10 @@
       }
     },
     "base": {
-      "dimension": {
+      "unit": {
         "value": "0.25rem",
         "type": "dimension",
-        "description": "Smallest unit in the baseline grid\n"
+        "description": "The base unit used in the grid system.\n"
       }
     },
```

Comment:
So as this name change, [this lines](https://github.com/Workday/canvas-tokens/blob/68b38be996ee7431065ad217b90b78d8bf4f0448/packages/canvas-tokens/utils/tokenStudioParser.ts#L16-L20) should be changed as well.

Reply:
this caused the build issue